docs/guides/high-availability/deploy-aws-accelerator-fencing-lambda.adoc

```diff
@@ -2,38 +2,40 @@
-To recover from the out-of-sync state a manual re-sync is necessary as described in <@links.ha id="operate-synchronize" />.
-This is why a site which is removed via fencing will not be re-added automatically when the network communication failure is resolved, but only after such a synchronisation using the manual procedure <@links.ha id="operate-site-online" />.
+In such scenarios, a quorum is commonly used to determine which sites are marked as online or offline.
+However, as multi-site deployments only consist of two sites, this is not possible.
+Instead, we leverage "`fencing`" to ensure that when one of the sites is unable to connect to the other site, only one site remains in the loadbalancer configuration, and hence only this site is able to serve subsequent users requests.
+
+In addition to the loadbalancer configuration, the fencing procedure disables replication between the two {jdgserver_name} clusters to allow serving user requests from the site that remains in the loadbalancer configuration.
+As a result, the sites will be out-of-sync once the replication has been disabled.
+
+To recover from the out-of-sync state, a manual re-sync is necessary as described in <@links.ha id="operate-synchronize" />.
+This is why a site which is removed via fencing will not be re-added automatically when the network communication failure is resolved, but only after such a synchronisation and using the manual procedure <@links.ha id="operate-site-online" />.
 
 In this {section} we describe how to implement fencing using a combination of https://prometheus.io/docs/alerting/latest/overview/[Prometheus Alerts]
-and AWS Lambda functions. A Prometheus Alert is triggered when split-brain is detected by the {jdgserver_name} server metrics,
-which results in the Prometheus AlertManager calling the AWS Lambda based webhook. The triggered Lambda function inspects
-the current Global Accelerator configuration and removes the site reported to be offline.
+and AWS Lambda functions.
+A Prometheus Alert is triggered when split-brain is detected by the {jdgserver_name} server metrics, which results in the Prometheus AlertManager calling the AWS Lambda based webhook.
+The triggered Lambda function inspects the current Global Accelerator configuration and removes the site reported to be offline.
 
-In a true split-brain scenario, where both sites are still up but network communication is down, it is possible that both
-sites will trigger the webhook simultaneously. We guard against this by ensuring that only a single Lambda instance can be executed at
-a given time.
+In a true split-brain scenario, where both sites are still up but network communication is down, it is possible that both sites will trigger the webhook simultaneously.
+We guard against this by ensuring that only a single Lambda instance can be executed at a given time.
```

Comment:
It's probably worth mentioning here that we also ensure that no action is taken if the number of members in the accelerator is only 1 as having a single Lambda instance alone is not sufficient to protect from the scenario you describe.